### PR TITLE
Call CanConvertFrom when a custom TypeConverterAttribute

### PR DESCRIPTION
**Pull request by leekelleher**
I've opened this as a PR, as I have one question... see inline comment below.

### CODE REVIEW

File: src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs [hunk 1]
```diff
@@ -344,7 +344,7 @@ private static object GetTypedProperty(IPublishedContent content, Type type)
                                     if (toConvert != null)
                                     {
                                         var converter = DependencyResolver.Current.GetService(toConvert) as TypeConverter;
-                                        if (converter != null)
+                                        if (converter != null && converter.CanConvertFrom(propertyType))
```
Comment (leekelleher) [inline src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs]: Question: What should we be passing to `CanConvertFrom`? `propertyType` or `propertyValue.GetType()`?

e.g. The intended type, or the type of the current value?

**Comment by leekelleher** [comment]
I think I've just answered my own question... it should be `propertyValue.GetType()`.

**Comment by mattbrailsford** [comment]
It should be the type of the current value.

**Comment by leekelleher** [comment]
Thanks @mattbrailsford. I realised that too when I was typing the question. Doh! :-)

You've got me thinking more about how `CanConvertTo` be used.

**Pull request merged**